package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "matter.js",
+  "version": "0.0.0",
```

Comment:
Perhaps 0.0.1?

Reply:
Discussed offline, 0.0.0 is to remove any expectation on npm that this contains something functional